jwcrypto/jwk.py

```diff
@@ -1052,6 +1052,22 @@ def __setattr__(self, item, value):
         except KeyError:
             raise AttributeError
 
+    @classmethod
+    def from_password(cls, password):
+        """Creates a symmetric JWK key from a user password.
+
+        :param key: A password in utf8 format.
+        """
+        obj = cls()
+        params = dict()
+        try:
+            params['kty'] = 'oct'
```

Comment:
You could leave this out of try-except, but doesn't really hurt to have it there as it should never fail anyway

Reply:
good point